src/services/favoritesService.js

```diff
@@ -0,0 +1,27 @@
+// import { StatusCodes } from 'http-status-codes';
+import favoriteModel from '../models/favorite';
+
+export const addToFavorite = async (data) => {
+  const { userId, houseId } = data;
+  if (!userId || !houseId) {
```

Comment:
It might better to handle this on the schema level by setting the fields as `required`.